client/cli/utils.go

```diff
@@ -0,0 +1,37 @@
+package cli
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/pflag"
+
+	"cosmossdk.io/x/upgrade/types"
+)
+
+// parsePlan was copied from the SDK because it was not exported
+// https://github.com/cosmos/cosmos-sdk/blob/x/upgrade/v0.1.4/x/upgrade/client/cli/parse.go
+func parsePlan(fs *pflag.FlagSet, name string) (types.Plan, error) {
+	height, err := fs.GetInt64(FlagUpgradeHeight)
+	if err != nil {
+		return types.Plan{}, err
+	}
+
+	info, err := fs.GetString(FlagUpgradeInfo)
+	if err != nil {
+		return types.Plan{}, err
+	}
+
+	return types.Plan{Name: name, Height: height, Info: info}, nil
+}
+
+// getDefaultDaemonName was copied from the SDK because it was not exported
+// https://github.com/cosmos/cosmos-sdk/blob/x/upgrade/v0.1.4/x/upgrade/client/cli/tx.go#L187
```

Comment:
Same as above.

```suggestion
// getDefaultDaemonName is copied from the Cosmos SDK because it is not exported.
//
// https://github.com/cosmos/cosmos-sdk/blob/x/upgrade/v0.1.4/x/upgrade/client/cli/tx.go#L184-L194
```